### DISCUSSION TITLE
Media - Radio Station and/or playlist

### DISCUSSION THREAD

**Discussion by Noiraux** [punxaphil/custom-sonos-card#84] [Q&A] [answered]
Hello, 

first thanks a lot for your usefull work!! that match perfectly what I want to do with my SONOS.

I'm looking for a way to play Radio Station? is it possible to add a media for a radio station? like a link with the URL (eg: http://scdn.nrjaudio.fm/adwz2/fr/30001/mp3_128.mp3?origine=fluxradios) ?

second question: how could I create a media containing a playlist on synology NAS, Itunes, deezer, spotify, ... ?

Nicolas.

**Comment by punxaphil** [comment, marked as answer]
There is already a radio browser built into HA, that one shows up in this card as well
<img width="1027" alt="image" src="https://user-images.githubusercontent.com/26582898/182401792-71dc9b85-51e9-4437-8704-aaf24bed8b6c.png">

---
PunxaPhil - I like [coffee](https://www.buymeacoffee.com/punxaphil) 🙂

**Reply by Noiraux** [reply]
Thanks a lot, I didn't noticed the [Radio Browser integration](https://www.home-assistant.io/integrations/radio_browser/). 
For my NAS Synology, I've installed [DLNA Server Integration](https://www.home-assistant.io/integrations/dlna_dms/) it works fine (Média folder, playlists, etc.)

Nicolas.
